code/modules/vehicles/armored/__armored.dm

```diff
@@ -75,6 +75,9 @@
 	///modifier to view range when manning a control chair
 	var/vis_range_mod = 0
 
+	var/obj/effect/turret_toss_holder/holder_obj
+	///the standard object that holds the sprite that gets tossed
```

Comment:
Lmao

Reply:
Actually why does this var exist you can just use a temp visual effect as you do anyway